otx/algorithms/classification/configs/configuration.yaml

```diff
@@ -297,14 +297,15 @@ algo_backend:
   description: parameters for algo backend
   header: Algo backend parameters
   train_type:
-    affects_outcome_of: NONE
+    affects_outcome_of: TRAINING
     default_value: INCREMENTAL
-    description: Quantization preset that defines quantization scheme
-    editable: false
```

Comment:
T/F values start with non-capital letter in these config yamls. I'm not sure if the parser of these files in geti case sensitive or not, but we can keep such values as non-capital at least for consittency.
Also, I'd like to point out that the yaml files normally are auto-generated from the BaseConfig class  (otx/algorithms/common/configs/training_base.py) and we don't need to modify them manually, but I forgot where is located the corresponding script in geti repo, so we can skip that step. Anyway, BaseConfig class should be modified accordingly (see section BaseAlgoBackendParameters).

Reply:
I know that we can revise it from baseconfig but some tasks like action does not support semi or other mode. I don't think that we need to modify the baseconfig. And capital or non-capital doesn't matter, which is not consistent already in same config file.

Reply:
Basically there should be no mess between python - yaml configs. If some parameters have different options for different tasks means that we should move them down in the python configs hierarchy. This may not be important for CLI, but in case of geti all the configs should be aligned. Since we don't use this branch in geti, let's merge it as is.
@supersoob could you provide the output of `otx train otx/algorithms/classification/configs/efficientnet_b0_cls_incr/template.yaml params -h`?

Reply:
Great. You made me think about geti side. It still requires remaining action task to be use only incremental so inconsistency is inevitable as of now. About inconsistency and capital aligning, I will write down it on my refactoring plan backlog. This is the output of the commands you gave.
![image](https://user-images.githubusercontent.com/37075645/207902325-1dc3f045-b27e-4a21-86e9-ff85b86642fd.png)
![image](https://user-images.githubusercontent.com/37075645/207902402-04948116-ffc3-46f4-bb9a-7c29f04eeb3d.png)